### DISCUSSION TITLE
Pick Exam date

### DISCUSSION THREAD

**Discussion by NorrisPau** [sebastian-hofstaetter/teaching#84] [General]
How do we pick which date we want to take the exam?

**Comment by sophiaalthammer** [comment]
Hi,
you can pick which ever date fits you the best.
If you fail the first exam, you can take the exam on the second date again.
I hope that answers your question!

**Reply by NorrisPau** [reply]
Thanks! I meant where we have to register for the dates. I could not see anything on TUWEL yet, will it open closely before the exam starts? :)

**Reply by sophiaalthammer** [reply]
The test will open on Tuwel at 12:00 midday and the test will be open for 24 hours. You dont have to sign up anywhere you just do one of the tests as long as they are available.

**Reply by ilijafilipovski** [reply]
Hi Sophia, 
Is there a limited number of tries for the test?

**Reply by sophiaalthammer** [reply]
Yes, you can try one time. But you have two tests, so if you fail the first one you can try in the second test again.

**Reply by sophiaalthammer** [reply]
Yes, you have 1 try for the test. If you fail the first test or if you cant take the first test, you can take the second test.

**Reply by repolly** [reply]
And where can we find the link for todays exam?
